config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -419,4 +422,36 @@ private OptionalInt getOptionalInt(final String key) {
       return JsonUtil.getBoolean(configRoot, key);
     }
   }
+
+  @Override
+  public List<Long> getForks() {
+    Stream<OptionalLong> forkBlockNumbers =
+        Stream.of(
+            getHomesteadBlockNumber(),
+            getDaoForkBlock(),
+            getTangerineWhistleBlockNumber(),
+            // duplicated for EIP155 and EIP158 handling
+            getSpuriousDragonBlockNumber(),
+            getSpuriousDragonBlockNumber(),
```

Comment:
is this meant to be duplicated?

Reply:
It shouldn't be duplicated.  Even if it is two separate EIPs it will be the same block number, and ForID will de-dup it.